regression/pages/whitelabel/const.py

```diff
@@ -0,0 +1,322 @@
+# -*- coding: utf-8 -*-
+"""
+List of constants to be used throughout the tests
+"""
+import os
+import uuid
+
+# Get HTTP Authentication credentials from environment variables
+AUTH_USERNAME = os.environ['BASIC_AUTH_USER']
+AUTH_PASSWORD = os.environ['BASIC_AUTH_PASSWORD']
+
+# Get API Access token
+
+ACCESS_TOKEN = os.environ['ACCESS_TOKEN']
+
+# Select the Org for which to run the tests, Default is MITProfessionalX
+ORG = os.getenv('ORG', 'MITProfessionalX')
```

Comment:
I'm confused we have this when we are setting the variables with paver separately. Seems redundant.